src/server/db_slice.cc

```diff
@@ -626,7 +634,10 @@ void DbSlice::FlushDbIndexes(const std::vector<DbIndex>& indexes) {
       if (db_ptr && db_ptr->stats.tiered_entries > 0) {
         for (auto it = db_ptr->prime.begin(); it != db_ptr->prime.end(); ++it) {
           if (it->second.IsExternal()) {
+            std::string tmp;
+            std::string_view key = it->first.GetSlice(&tmp);
             PerformDeletion(it, shard_owner(), db_ptr.get());
+            SendInvalidationTrackingMessage(key);
```

Comment:
why dont you make PerformDeletion function of DbSlice? I dont like it that we need to remember to call SendInvalidationTrackingMessage whenever we call PerformDeletion

Reply:
ok, will do.

Reply:
> why dont you make PerformDeletion function of DbSlice? I dont like it that we need to remember to call SendInvalidationTrackingMessage whenever we call PerformDeletion

done, please take another look. I like this change, saved a lot of code duplication.